pkg/istio-agent/sds-agent.go

```diff
@@ -371,6 +389,31 @@ func (sa *SDSAgent) newSecretCache(serverOptions sds.Options) (workloadSecretCac
 		// used.
 		caClient, err = gca.NewGoogleCAClient(serverOptions.CAEndpoint, true)
 		serverOptions.PluginNames = []string{"GoogleTokenExchange"}
+	} else if serverOptions.CAProviderName == keyfactorCAName {
+		// Assume CA from Keyfactor is mounting /etc/certs
```

Comment:
How does this work with SDS?

Reply:
When user change caProvider to "KeyfactorCA" on global values.

Reply:
how or who is mounting the cert in /etc/certs?

Reply:
![image](https://user-images.githubusercontent.com/49465776/83476320-9c37db00-a4ba-11ea-9d20-33c53c9d8a87.png)
I found a secret in the injection template. It's mounted into path "/etc/certs". And I reference to implement of Citadel

Reply:
That's from legacy implementation. Citadel doesn't exist anymore